ui.frontend/webpack.dev.js

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+const path = require('path');
+
+module.exports = {
+    mode: 'development',
+    devtool: 'source-map',
+    entry: './src/index.js',
+    output: {
+        filename: 'main.js',
+        path: path.resolve(__dirname, 'dist'),
+        clean: true,
+        library: {
+            name: 'FormView',
+            type: 'window'
+        },
+    },
```

Comment:
can you have a look at the archetype project. We can share configs between prod and dev webpack configs so that things can be shared

Reply:
done. although optimisation can come later after this first cut.